app/views/feedback/steps/_rating.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= f.govuk_text_area :unsuccessful_visit_explanation, label: { text: "Tell us about your experience signing up for #{feedback_session['subject']}"} do %>
+<p>
+  You can give general feedback or tell us about a specific problem.
+  Do not include any information that could identify you personally, such as your name.
+</p>
+<% end %>
+
+<%= f.govuk_collection_radio_buttons :rating,
```

Comment:
Ideally this question would go on a separate page @StackedPancakez - is this possible?

Reply:
As discussed - we would like to separate out the free text question and the radio button satisfaction question onto separate steps for all segments (website, mailing list, adviser, events)